qcog_python_client/qcog.py

```diff
@@ -549,6 +591,11 @@ def status(self) -> TrainingStatus:
         )
 
         self.last_status = TrainingStatus(self.status_resp["status"])
+
+        # Try to set the _loss attribute if present
+        if self.status_resp.get("loss"):
+            self._loss = self.status_resp["loss"]
```

Comment:
what do you think of the following?
```
        if loss := self.status_resp.get("loss"):
            self._loss = loss
```

Reply:
honestly, `_loss` can be `None`, I can just directly assign the value from the `get` method